packages/landing-site/src/components/contributor-banner.tsx

```diff
@@ -0,0 +1,18 @@
+import { ContributorBanner as _ContributorBanner } from "@framework/system/src/components/homepage/contributor-banner"
+import { ContributorData } from "@framework/system/src/components/homepage/contributor"
+import { sprinkles } from "@framework/system/src/sprinkles/sprinkles.css"
+
+export default function ContributorBanner({
+	contributors,
+}: {
+	contributors: ContributorData[]
+}) {
+	return (
+		<_ContributorBanner
+			contributors={contributors}
+			className={sprinkles({
+				marginTop: { mobile: 56, desktop: 152 },
```

Comment:
This might be a nitpick but noticed in the `resources-info-banner` component, the margin on mobile was essentially 1/2 that of desktop. In this case, there is no clear correlation between mobile and desktop. 

Imo, we could pick a multiple of the mobile margin to be the desktop margin. That way when someone is picking margins to add, they have something to base off.

Thoughts @ktrz ?

Reply:
I can't specify different values than what `sprinkles` allows me to. I just picked the closest values that made sense
![Screenshot 2022-12-16 at 16 02 35](https://user-images.githubusercontent.com/22615575/208127908-63185e1f-bbe4-48e7-adc7-c821a16ff7d7.png)

Reply:
Ahh, that makes sense. 👍🏾